vulture/utils.py

```diff
@@ -46,10 +47,11 @@ def condition_is_always_true(condition):
 
 
 def format_path(path):
-    if not path:
-        return path
-    relpath = os.path.relpath(path)
-    return relpath if not relpath.startswith("..") else path
+    path_str = str(path)
+    if not path_str:
+        return path_str
+    relpath = os.path.relpath(path_str)
```

Comment:
Is there no pathlib alternative for this?

Reply:
There is relative_to but docs said they were not exactly equivalent. But it seems to be enough for our purpose. Has added it.